src/GraphQL/Validation/ValidationContext.cs

```diff
@@ -191,7 +195,7 @@ public List<FragmentDefinition> GetRecursivelyReferencedFragments(Operation oper
         /// <summary>
         /// Returns all of the variable values defined for the operation from the attached <see cref="Variables"/> object.
         /// </summary>
-        public Variables GetVariableValues(ISchema schema, VariableDefinitions? variableDefinitions, Inputs variables, IVariableVisitor? visitor = null)
+        public Variables GetVariableValues(VariableDefinitions? variableDefinitions, IVariableVisitor? visitor = null)
```

Comment:
Schema and Inputs are already into `ValidationContext`.

Reply:
Operation should be stored in the context, and VariablesDefinitions should be pulled from Operation.Variables (see suggestions below)